mps-model-adapters-plugin/build.gradle.kts

```diff
@@ -33,8 +33,8 @@ tasks {
     if (mpsPluginDir != null && mpsPluginDir.isDirectory) {
         create<Sync>("installMpsPlugin") {
             dependsOn(prepareSandbox)
-            from(project.layout.buildDirectory.dir("idea-sandbox/plugins/mps-model-server-plugin"))
-            into(mpsPluginDir.resolve("mps-model-server-plugin"))
+            from(project.layout.buildDirectory.dir("idea-sandbox/plugins/mps-model-adapters-plugin"))
+            into(mpsPluginDir.resolve("mps-model-adapters-plugin"))
```

Comment:
Was this an independent bug? Or did the mps-model-server-plugin include the model-adapters-plugin previously?

Reply:
Independent bug.
See c6d49008cff803af01f1f6a7d557f7779c31e4db